tasks/request_files/request_files.py

```diff
@@ -219,12 +226,14 @@ def inner_task(event: Dict, max_retries: int, retry_sleep_secs: float,
             file_key = keys[FILE_KEY_KEY]
             destination_bucket_name = keys[FILE_DEST_BUCKET_KEY]
             if object_exists(s3, glacier_bucket, file_key):
```

Comment:
We should double check to see if a card is written, in future iterations, we should flush this out more possibly in the https://github.com/nasa/cumulus-orca/blob/develop/tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py code where we prefilter based on the users configured excludedFileTypes value. Allowing us to better articulate to the end user when a file does not exist if it is a configuration information item or an actual error where data that should be in ORCA is actually missing.

Reply:
created card for this task:
https://bugs.earthdata.nasa.gov/browse/ORCA-207